cfgov/unprocessed/js/routes/browse-basic/index.js

```diff
@@ -0,0 +1,26 @@
+/* ==========================================================================
+   Scripts for /browse-basic/.
+   ========================================================================== */
+
+'use strict';
+
+// Required polyfills for <IE8.
+require( '../../modules/polyfill/query-selector' );
+
+// List of organisms used.
+var Expandable = require( '../../molecules/Expandable' );
+var ExpandableGroup = require( '../../organisms/ExpandableGroup' );
+
+var expandables = document.querySelectorAll( '.expandable-prototypes .m-expandable' );
```

Comment:
Because expandables can appear on their own or within expandable groups. When they appear in an expandable group, the expandable group handles initialization of the expandables it contains. So this scopes the query only to the lone expandables. In L:21 `.expandable-group-prototypes` could be removed, but is provided just for consistency.